modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/common/move_kb_object_modal.jsp

```diff
@@ -28,7 +28,7 @@ KBAdminNavigationDisplayContext kbAdminNavigationDisplayContext = new KBAdminNav
 			HashMapBuilder.<String, Object>put(
 				"items", kbAdminNavigationDisplayContext.getKBFolderDataJSONArray()
 			).put(
-				"itemToMoveId", ParamUtil.getString(request, "itemToMoveId")
+				"itemToMoveParent", kbAdminNavigationDisplayContext.getKBObjectToMoveParent()
```

Comment:
Shouldn't this be `kbObjectToMoveParentKBObjectId`?

Reply:
I will leave it as `getKBObjectToMoveParentId()`, and in frontend side `itemToMoveParentId` because it is a generic component